packages/devtools_app/lib/src/framework/release_notes/release_notes.dart

```diff
@@ -52,6 +52,10 @@ class ReleaseNotesController extends SidePanelController {
     _init();
   }
 
+  @visibleForTesting
+  static Uri get releaseIndexUrl =>
```

Comment:
sytle-nit: remove type on LHS

Reply:
Since this is a getter, doing so will trigger [`always_declare_return_types`](https://dart.dev/tools/linter-rules/always_declare_return_types).